docs/hugo-watch-mode-setup.md

```
@@ -0,0 +1,349 @@
+
+# Follow file watcher logs
+./bin/watch-compose logs-watcher -f
+```
+
+### 4. Development Workflow
+
+1. Make small changes (≤3 lines for micro-refactoring)
+2. File watcher detects changes automatically
+3. Selective tests run based on changed files
+4. Hugo rebuilds content instantly
+5. View results at http://localhost:1313
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)